### PR TITLE
Feature/deal

### PR DESCRIPTION
**Pull request by ZelikSV**

### CODE REVIEW

File: src/actions/converter.actions.js [hunk 1]
```diff
@@ -48,6 +51,23 @@ export const swappingCurrency = () => ({
...
 export const sendCurrencyTransaction = value => ({
-  type: SEND_DIAL,
-  payload: value,
+  type: SEND_DEAL,
+  payload: { ...value },
+});
+
+export const sendCurrencyTransactionPost = () => ({
+  type: SEND_DEAL_POST,
+  onSending: true,
```
Comment (motifej) [inline src/actions/converter.actions.js]: onSending: true - just in reducer use it, not here

File: src/sagas/currencySaga/index.js [hunk 2]
```diff
@@ -37,10 +41,44 @@ function* countCurrencies(action) {
   yield put(countCurrencyAsync(buyValue));
 }
 
+function* sendDealOfConverting(action) {
+  const {
```
Comment (motifej) [inline src/sagas/currencySaga/index.js]: left all data in payload

File: src/sagas/currencySaga/index.js [hunk 3]
```diff
@@ -37,10 +41,44 @@ function* countCurrencies(action) {
...
+    currencySell,
+    amountSell,
+    currencyBuy,
+    amountBuy,
+    fee,
+  } = action.payload;
+
+  yield put(sendCurrencyTransactionPost());
+  try {
+    yield call(
```
Comment (motifej) [inline src/sagas/currencySaga/index.js]: yield call(createDealInDB, action.payload)

File: src/actions/converter.actions.js (hunk 1 above)
Comment (ZelikSV) [reply src/actions/converter.actions.js]: have done

File: src/sagas/currencySaga/index.js (hunk 2 above)
Comment (ZelikSV) [reply src/sagas/currencySaga/index.js]: Ok

File: src/actions/converter.actions.js [hunk 4]
```diff
@@ -48,6 +48,20 @@ export const swappingCurrency = () => ({
 });
 
 export const sendCurrencyTransaction = value => ({
-  type: SEND_DIAL,
-  payload: value,
+  type: SEND_DEAL,
+  payload: { ...value },
```
Comment (motifej) [inline src/actions/converter.actions.js]: payload: value

File: src/sagas/currencySaga/index.js (hunk 3 above)
Comment (ZelikSV) [reply src/sagas/currencySaga/index.js]: Already have done

File: src/actions/converter.actions.js (hunk 4 above)
Comment (ZelikSV) [reply src/actions/converter.actions.js]: ok

File: src/sagas/currencySaga/index.js [hunk 5]
```diff
@@ -37,10 +41,25 @@ function* countCurrencies(action) {
   yield put(countCurrencyAsync(buyValue));
 }
 
+function* sendDealOfConverting(action) {
+  yield put(sendCurrencyTransactionPost());
+  try {
+    console.log(action.payload);
```
Comment (AlexYalovoy) [inline src/sagas/currencySaga/index.js]: get rid of it

Comment (ZelikSV) [reply src/sagas/currencySaga/index.js]: Thank you.

**Pull request merged**